uefi/README.md

```diff
@@ -11,6 +11,15 @@ This crate makes it easy to develop Rust software that leverages **safe**,
 ![Build status](https://github.com/rust-osdev/uefi-rs/workflows/Rust/badge.svg)
 ![Stars](https://img.shields.io/github/stars/rust-osdev/uefi-rs)
 
+## Value-add and Use Cases
+
+With `uefi`, you have the flexibility to integrate selected types and
+abstractions into your project or to conveniently create EFI images,
+addressing the entire spectrum of your development needs. It supports you
```

Comment:
nit: I would drop "addressing the entire spectrum of your development needs", that seems like maybe too bold of a claim :)

(Like maybe the Qt library could make a claim like that, since they have utilities for practically every use case under the sun, but uefi-rs is a bit more focused.)